### ISSUE TITLE
Parsing error on Sass calculation

### ISSUE THREAD

**Issue by shaneog** [sasstools/sass-lint#210]
The following Scss code causes a parsing error:
```shell
/usr/local/lib/node_modules/sass-lint/lib/groot.js:21
    throw new Error('Parsing error' + fileInfo + ': ' + e.message);
    ^

Error: Parsing error at test.scss: Please check validity of the block starting from line #9
    at module.exports (/usr/local/lib/node_modules/sass-lint/lib/groot.js:21:11)
    at Function.sassLint.lintText (/usr/local/lib/node_modules/sass-lint/index.js:34:13)
    at /usr/local/lib/node_modules/sass-lint/index.js:87:21
    at Array.forEach (native)
    at Function.sassLint.lintFiles (/usr/local/lib/node_modules/sass-lint/index.js:86:9)
    at detectPattern (/usr/local/lib/node_modules/sass-lint/bin/sass-lint.js:16:18)
    at /usr/local/lib/node_modules/sass-lint/bin/sass-lint.js:58:5
    at Array.forEach (native)
    at Object.<anonymous> (/usr/local/lib/node_modules/sass-lint/bin/sass-lint.js:57:16)
    at Module._compile (module.js:434:26)
```
**Versions:**
```shell
$ sass-lint --version
1.2.1
$ node --version
v4.1.0
$ npm --version
3.3.3
```
**Test Case:**  `sass-lint -v test.scss`
```scss
$radius: 45;
$pi: 3.141592653589793;

// Outer container
.radial {
  @for $j from 0 through 100 {
    &.filled_#{$j} {
      .filled {
        stroke-dashoffset: #{((100 - $j) / 100) * ($pi * ($radius * 2))};
      }
    }
  }
}

```

**Comment by DanPurdy** [comment]
Unfortunately we're awaiting  release of our AST which will fix a problem with passing interpolation. `#{$j}` currently doesn't parse...

**Comment by DanPurdy** [comment]
Hi there @shaneog I've just run this through the latest version of the AST we use gonzales-pe (3.2.1) and this is still unfortunately causing a parse error. I've filed it within an issue that you can find [here](https://github.com/tonyganch/gonzales-pe/issues/115#issuecomment-149358821)

**Comment by shaneog** [comment]
Thanks @DanPurdy!

**Comment by DanPurdy** [comment]
@shaneog This is fixed in sass-lint 1.6.0
